dags/dags_by_country.py

```diff
@@ -0,0 +1,36 @@
+from airflow import DAG
```

Comment:
Is the intention to have a collection of "per-country" dags and a collection of "per-dataset" dags? And then allow the user to select whichever option is most suitable for them? I like this in principle. At some point in the future, we might need to think about how this appears to the user in the Airflow environment.

Reply:
Absolutely correct! Previously, it was possible to schedule only the dags related to some specific data type (like "give me all railways for all the countries we have"). That is unusable in the emergency context, though, since you usually need the data for some specific country.

Reply:
👍